app/controllers/users_controller.rb

```diff
@@ -5,5 +5,7 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @posts = @user.posts.includes(comments: [:author])
+    @post = Post.includes(comments: [:author]).find(params[:id])
```

Comment:
- [ ] Good job so far 👏 👏 As suggested by the previous reviewer, please fix the error that is shown up when someone tries to view the user. When someone tries to click on the `show this user` the error below is presented 👇 
![image](https://user-images.githubusercontent.com/90222110/215273260-08367742-6190-4e08-aac9-96811f029ec1.png)

Kindly fix that 🙏

Reply:
@BertrandConxy Idk. Its working on my side. 

![Screenshot from 2023-01-28 20-00-54](https://user-images.githubusercontent.com/57269872/215273525-bf9a3219-ff55-4f40-9ce0-115c43f31625.png)
![Screenshot from 2023-01-28 20-00-49](https://user-images.githubusercontent.com/57269872/215273532-53e4833a-701e-4adc-a83e-8d4c9ca8567d.png)

Reply:
@BertrandConxy @cynthiainga The controller action you both have highlighted has nothing to do with the route to the user view.